crates/project/src/dap_store.rs

```diff
@@ -170,6 +173,10 @@ impl DapStore {
         upstream_client: AnyProtoClient,
         _: &mut ModelContext<Self>,
     ) -> Self {
+        upstream_client
```

Comment:
Can we move this to the `shared` method? This feels a bit hacky solution. So we also don't try to send a proto message when you are not event sharing a project.

Reply:
Yes